shell.nix

```diff
@@ -28,6 +28,7 @@ let
     hooks = {
       purty.enable = true;
       stylish-haskell.enable = true;
+      terraform-format.enable = true;
```

Comment:
Do we need to pass in our terraform above to ensure it's using the same version as we have pinned?

Reply:
Oh, yes! thank you for noticing!

Reply:
Actually i'll have to leave it like this for now because: 

terraform isn't being used directly. Instead it uses a wrapper around terraform that can deal with getting passed multiple files as arguments: https://github.com/cachix/pre-commit-hooks.nix/blob/master/nix/terraform-fmt/default.nix

So right now i could only overwrite the whole wrapper and not terraform itself. It's not ideal but i don't expect them to change formatting much and i'll address this upstream instead.